sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -344,23 +344,7 @@ impl OpenOcdServer {
     // Convert a RISC-V register name into a name that can be used
     // with read_register32.
     fn riscv_reg_name(&self, reg: &RiscvReg) -> &'static str {
-        match reg {
-            RiscvReg::GprByName(gpr) => match gpr {
-                RiscvGpr::A0 => "a0",
-                RiscvGpr::GP => "gp",
-                RiscvGpr::SP => "sp",
-                RiscvGpr::RA => "ra",
-            },
-            RiscvReg::CsrByName(csr) => match csr {
-                RiscvCsr::DPC => "dpc",
-                RiscvCsr::PMPCFG0 => "pmpcfg0",
-                RiscvCsr::PMPCFG1 => "pmpcfg1",
-                RiscvCsr::PMPCFG2 => "pmpcfg2",
-                RiscvCsr::PMPCFG3 => "pmpcfg3",
-                RiscvCsr::PMPADDR0 => "pmpaddr0",
-                RiscvCsr::PMPADDR15 => "pmpaddr15",
-            },
-        }
+        (*reg).into()
     }
```

Comment:
Do you think we should keep this method if it's just calling `Into::into`?

Reply:
I like the explicitness of a function that does `RiscvReg` -> &'static str` conversion. But having this in `openocd.rs` here does seem a bit weird.

I moved this to become an associated function of `RiscvReg`. PTAL

Reply:
I'm happy with that, thanks!